oneflow/user/kernels/distributions/uniform_int_kernel.h

```diff
@@ -76,9 +76,10 @@ class UniformIntKernel final : public user_op::OpKernel {
   std::shared_ptr<user_op::OpKernelState> CreateOpKernelState(
       user_op::KernelInitContext* ctx) const override {
     const auto& generator = CHECK_JUST(one::MakeAutoGenerator());
-    /* each rank get a different seed to generate the same distribution,
-      but different values of local tensor */
-    generator->set_current_seed(CHECK_JUST(GetOpKernelRandomSeedInCurrentRank(ctx, ctx->Attr<int64_t>("seed"))));
+    // Each rank get seed depend on SBP,when SBP is Spit, use different seeds,when SBP is B and P,
+    // use the same seed;
```

Comment:
// Each rank get seed depend on SBP. When SBP is Spit, use different seeds; when SBP is B and P,
    // use the same seed.